### PR TITLE
[frontend] Improve PolarArea widget

### LINKED ISSUE #6520: PolarArea widget does not take into account the colors of the entities
Horizontal bars and Donut widgets , when they show entities which have a color, such as Markings, Labels or Status, use the color defined inside the entity data.

image

This is not the case for the PolarArea, as shown below

image

### LINKED ISSUE #6521: Widgets that use the entity color might end up unreadable
If you create a Donut widget on markings, the widget will use the color of the markings as colors inside the chart.

Howevre, the labels in the tooltips are rendered on top of this fixed color, and it might end up white-on-white which is unreadable.

image

### PR DESCRIPTION
**Pull request by lndrtrbn**
### Proposed changes

* use colors when possible
* improve tooltip to avoid white on white colors

### Related issues

* #6520 
* #6521 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by SamuelHassine** [comment]
@lndrtrbn can you also make polar area available for "entities" perspective as the other distribution widgets? It is not normal that it is only available in "knowledge graph" perspective. Check the donut behavior for comparison, it should be the same.

**Review by marieflorescontact — COMMENTED** [review]
locally tested ok for `relastionship` perspective 👌

File: opencti-platform/opencti-front/src/components/dashboard/WidgetPolarArea.tsx [hunk 1]
```diff
@@ -4,29 +4,38 @@ import { useTheme } from '@mui/styles';
...
 import type { Theme } from '../Theme';
+import useDistributionGraphData from '../../utils/hooks/useDistributionGraphData';
 
 interface WidgetPolarAreaProps {
-  data: {
-    label: string,
-    value: number
-  }[]
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any[]
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/components/dashboard/WidgetPolarArea.tsx]: I'm curious; does it work if you use DistributionQueryData from useDistributionGraphData ?

Comment (lndrtrbn) [reply opencti-platform/opencti-front/src/components/dashboard/WidgetPolarArea.tsx]: I'll check thanks!

Comment (lndrtrbn) [reply opencti-platform/opencti-front/src/components/dashboard/WidgetPolarArea.tsx]: Done

Comment (labo-flg) [reply opencti-platform/opencti-front/src/components/dashboard/WidgetPolarArea.tsx]: great !

**Review by labo-flg — APPROVED** [review]

**Pull request merged**